_translator-files/po/de/Installation-for-iOS.po

```diff
@@ -3,149 +3,150 @@
-msgstr ""
+msgstr "## Installation für iOS (nur für erfahrene Benutzer) **Hinweis**: Wir stellen eine unsignierte .ipa-Datei zur Verfügung, die du signieren und dann auf deinem Gerät installieren musst. Wenn dein Gerät nicht Jailbroken ist (und AppSync installiert hast), benötigst du möglicherweise zusätzliche Software, um Jamulus zu installieren. Dafür gibt es mehrere Möglichkeiten, aber in dieser Anleitung wird nur die gängigste Methode beschrieben. Bitte beachte, dass wir davon ausgehen, dass du bereits ein (kostenloses) Apple-Entwicklerkonto hast. Wenn du noch keines hast, erstelle eine neue Apple ID und [registriere dich für ein Entwicklerkonto](https://developer.apple.com/membercenter). Wir empfehlen dir, ein Konto ohne Zwei-Faktor-Authentifizierung zu erstellen, das nicht mit deinem persönlichen Konto verknüpft ist. Du musst das Passwort in die Nicht-Apple-Software eingeben."
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-iOS.md:28
 msgid "[Download Jamulus (.ipa) on a PC/Mac]({{ site.download_root_link }}{{ site.download_file_names.ios }}){:.button}"
-msgstr ""
+msgstr "[Download Jamulus (.ipa) on a PC/Mac]({{ site.download_root_link }}{{ site.download_file_names.ios }}){:.button}"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-iOS.md:29
 msgid "Sign the .ipa file (e.g. via Xcode or some other **trusted** app. We won't give recommendations here, but there exist apps for every operating system)"
-msgstr ""
+msgstr "Signiere die .ipa-Datei (z. B. mit Xcode oder einer anderen **vertrauenswürdigen** Anwendung. Wir werden hier keine Empfehlungen geben, aber es gibt Apps für jedes Betriebssystem)"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-iOS.md:30
 msgid "Once signed and installed; on your iOS device open Settings, navigate to General>Profiles (or VPN and Device Management). Tap on the developer account corresponding to your account and trust the account."
-msgstr ""
+msgstr "Nach der Unterzeichnung und Installation öffnest du auf deinem iOS-Gerät die Einstellungen und navigierst dann zu Allgemein>Profile (oder VPN und Geräteverwaltung). Tippe auf das Entwicklerkonto, das deinem Konto entspricht, und vertraue dem Konto."
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-iOS.md:31
 msgid "Open Jamulus from your home-screen"
-msgstr ""
+msgstr "Öffne Jamulus über den Startbildschirm"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-iOS.md:32
 msgid "After 7 days you will need to re-sign Jamulus if you are using a free developer account. Re-do everything from Step 4."
-msgstr ""
+msgstr "Nach 7 Tagen musst du dich erneut bei Jamulus anmelden, wenn du ein kostenloses Entwicklerkonto verwendest. Führe die Aktivierung ab Schritt 4 erneut durch."
```

Comment:
No. Not logging in to Jamulus but re-signing the app:

```suggestion
msgstr "Nach 7 Tagen musst du Jamulus neu signieren, wenn du ein kostenloses Entwicklerkonto verwendest. Führe die alles ab Schritt 4 erneut durch."
```

Reply:
Corrected